cmd/operator-sdk/main.go

```diff
@@ -42,6 +47,17 @@ func main() {
 		Use:     "operator-sdk",
 		Short:   "An SDK for building operators with ease",
 		Version: osdkversion.Version,
+		PersistentPreRun: func(cmd *cobra.Command, args []string) {
+			if viper.GetBool(flags.VerboseOpt) {
+				err := projutil.SetGoVerbose()
+				if err != nil {
+					log.Errorf("Could not set GOFLAGS: (%v)", err)
```

Comment:
Error msgs should start with lowercase:

```suggestion
					log.Errorf("could not set GOFLAGS: (%v)", err)
```